sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/properties/utils/AzureEventHubsKafkaPropertiesUtils.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.autoconfigure.implementation.properties.utils;
+
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
+
+/**
+ * Utils class for {@link KafkaProperties} String constants.
+ */
+public final class AzureEventHubsKafkaPropertiesUtils {
+
+    private AzureEventHubsKafkaPropertiesUtils() {
+
+    }
+
+    public static final String SECURITY_PROTOCOL_CONFIG_SASL = "SASL_SSL";
```

Comment:
how about consider using the original const in `kafka-client` lib.
for example:
SecurityProtocol.SASL_SSL 
OAuthBearerLoginModule.OAUTHBEARER_MECHANISM

Reply:
good catch, updated in #29404